packages/@roots/bud-server/src/middleware/proxy/proxy.options.ts

```diff
@@ -49,8 +49,10 @@ export class OptionsFactory {
       [url.proxy.host]: url.dev.host,
     }
 
+    this.options.hostRewrite = url.dev.host
     this.options.onProxyReq = request
     this.options.onProxyRes = interceptor
+    this.options.secure = false
```

Comment:
ssl compatible just means that the proxy middleware won't attempt to validate the certs

Reply:
I was running into issues with this last night. Might still need that one environment variable set to make HTTPS work correctly. Maybe another PR tho idk.